woocommerce/woocommerce-android
# Show onboarding error when pending requirements present

Closes: #7229


### Description

The PR adds a specific error message when stripe pending requirements present

### Testing instructions

* if you have a site with pending requirements then skip 2 and use this site
* Change the code and return `StripeAccountPendingRequirement` from `CardReaderOnboardingChecker::getOnboardingState`
* Go to More -> Payments
* Notice error message
* Notice that all buttons are enabled and clickable
* Click on the error message and notice error screen

### Images/gif


https://user-images.githubusercontent.com/4923871/185558414-10e66c52-6662-40af-a137-56dfd4076d17.mp4
